wnbd-client/client.cpp

```diff
@@ -0,0 +1,333 @@
+#include "client.h"
+#include "cmd.h"
+#include "usage.h"
+#include "wnbd.h"
+#include "version.h"
+
+#include <boost/exception/diagnostic_information.hpp>
+
+#include <iostream>
+#include <iomanip>
+
+namespace po = boost::program_options;
+using namespace std;
+
+vector<Client::Command*> Client::commands;
+
+Client::Command* Client::get_command(string name) {
```

Comment:
If we want to follow the current project formatting the `{` curly bracket should be on a new line.